databases/week3/meal_sharing.sql

```diff
@@ -1,33 +1,461 @@
+	`location`,
+	`when`,
+	`max_reservations`,
+	`price`,
+	`created_date`
+) VALUES (
+	"Vegan brunch",
+	"Oatmeal grød with almond milk",
+	"Vesterbro",
+	"2019-11-27",
+	2,
+	99,
+	"2019-11-27"
+);
+
+INSERT INTO `meal` (
+	`title`,
+	`description`,
+	`location`,
+	`when`,
+	`max_reservations`,
+	`price`,
+	`created_date`
+) VALUES (
+	"Turkish brunch",
+	"Including 'sucuklu yumurta' and 'börek'",
+	"Nørrebro",
+	"2019-12-03",
+	4,
+	119,
+	"2019-11-26"
+);
+
+INSERT INTO `meal` (
+	`title`,
+	`description`,
+	`location`,
+	`when`,
+	`max_reservations`,
+	`price`,
+	`created_date`
+) VALUES (
+	"Turkish brunch",
+	"Including 'sucuklu yumurta' and 'börek'",
+	"Nørrebro",
+	"2019-12-02",
+	4,
+	119,
+	"2019-11-25"
+);
+
+INSERT INTO `meal` (
+	`title`,
+	`description`,
+	`location`,
+	`when`,
+	`max_reservations`,
+	`price`,
+	`created_date`
+) VALUES (
+	"Vegan brunch",
+	"Oatmeal grød with almond milk",
+	"Vesterbro",
+	"2019-12-02",
+	4,
+	99,
+	"2019-11-25"
+);
+
+INSERT INTO `meal` (
+	`title`,
+	`description`,
+	`location`,
+	`when`,
+	`max_reservations`,
+	`price`,
+	`created_date`
+) VALUES (
+	"Danish brunch",
+	"Sausages, omlette and pancakes",
+	"Østerbro",
+	"2019-12-04",
+	4,
+	120,
+	"2019-11-25"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	2,
+	2,
+	"2019-11-27"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	2,
+	3,
+	"2019-11-27"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	4,
+	1,
+	"2019-11-25"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	2,
+	2,
+	NOW()
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	4,
+	2,
+	"2019-12-01"
+);
+
+INSERT INTO `reservation` (
+	`number_of_guests`,
+	`meal_id`,
+	`created_date`
+) VALUES (
+	4,
+	3,
+	"2019-12-01"
+);
+
+INSERT INTO `review` (
+	`title`,
+	`description`,
+	`meal_id`,
+	`stars`,
+	`created_date`
+) VALUES (
+	"Expensive",
+	"Taste was ok but they charge too much for an oat meal",
+	3,
+	2,
+	NOW()
+);
+
+INSERT INTO `review` (
+	`title`,
+	`description`,
+	`meal_id`,
+	`stars`,
+	`created_date`
+) VALUES (
+	"Nice",
+	"We liked the taste of everything",
+	2,
+	4,
+	NOW()
+);
+
+INSERT INTO `review` (
+	`title`,
+	`description`,
+	`meal_id`,
+	`stars`,
+	`created_date`
+) VALUES (
+	"Amazing",
+	"The best brunch I have ever had",
+	1,
+	5,
+	NOW()
+);
+
+INSERT INTO `review` (
+	`title`,
+	`description`,
+	`meal_id`,
+	`stars`,
+	`created_date`
+) VALUES (
+	"Average",
+	"It was alright but maybe I had bigger expectations and it couldn't fulfill them.",
+	2,
+	3,
+	NOW()
+);
+
+-- [FUNCTIONALITY]
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT * FROM meal
+WHERE price < 100;
+
+-- Get meals that still has available reservations
+SELECT * FROM meal
+WHERE `when` > "2019-11-27";
```

Comment:
This one is a bit tricky to implement. I can try to give a few hints. You need to join the reservation table and compare the total number of guests in the reservations to the maximum allowed number of guests for a meal. Hope that helps :)